setup-scripts/README.md

```diff
@@ -82,16 +81,12 @@ To enable the 2nd service: `systemctl --user enable siad-upload.service`
 
 ### Useful Commands
 
-To start the service: `systemctl --user start siad`
-
-To stop it: `systemctl --user stop siad`
-
-To check the status of it: `systemctl --user status siad`
-
-To check standard err/standard out: `journalctl --user-unit siad`. In addition you can add:
-
-- `-r` to view journal from the newest entry
-- `-f` to follow and `-n INTEGER` to specify number of lines
+- To start the service: `systemctl --user start siad`
+- To stop it: `systemctl --user stop siad`
+- To check the status of it: `systemctl --user status siad`
+- To check standard err/standard out: `journalctl --user-unit siad`. In addition you can add:
```

Comment:
You probably need to be root for these.

Reply:
nope it works fine because siad is ran as `user`